plugins/node/opentelemetry-instrumentation-graphql/test/graphql.test.ts

```diff
@@ -976,7 +979,7 @@ describe('graphql', () => {
       assert.deepStrictEqual(spans.length, 1);
     });
 
-    it('should instrument parse with error', () => {
+    itMaybe('should instrument parse with error', () => {
```

Comment:
This test doesn't work on Node.js v8 and Node.js v10 when using `graphql` v16, so make only run it on more recent versions of Node.js.